rancid-tomatillos/src/Components/MovieView/MovieView.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./MovieView.css"
+
+const MovieView = ({selectedMovie}) => {
+
```

Comment:
A small refactor you could do here is another layer of destructuring.  For example:

```
const { title, backdrop_path, release_date, tagline, average_rating } = selectedMovie;

return (
  <h2>{title}</h2>
)
```

I'm shorthanding it a bit, but hopefully that makes sense!